docs/configuration/miscellaneous.md

```diff
@@ -180,6 +180,14 @@ When determining the primary IP address for a device, IPv6 is preferred over IPv
 
 ---
 
+## PUBLIC_SWAGGER
+
+Default: True
+
+Swagger exposes by default the schemas to unauthenticated users. Set this to False to only expose them to authenticated users.
```

Comment:
This probably needs a bit more context, as the casual reader might not know what Swagger is or where the schemas can be accessed. (Feel free to link to other areas of the documentation.)